typescript/lib/config.ts

```diff
@@ -701,7 +709,10 @@ export class AIConfigRuntime implements AIConfig {
         );
       }
 
-      prompt.metadata[key] = value;
+      prompt.metadata = {
+        ...prompt.metadata,
```

Comment:
does the spread operator automatically ignore optionals if it doesn't exist?

Reply:
Yep!
<img width="644" alt="Screenshot 2023-11-22 at 6 21 58 PM" src="https://github.com/lastmile-ai/aiconfig/assets/5060851/53658d9d-dbed-48de-908a-6ce81c537895">